pandas/core/generic.py

```diff
@@ -10859,7 +10859,15 @@ def _accum_func(
         def block_accum_func(blk_values):
             values = blk_values.T if hasattr(blk_values, "T") else blk_values
 
-            result = nanops.na_accum_func(values, func, skipna=skipna)
+            from pandas.core.construction import ensure_wrapped_if_datetimelike
+
+            values = ensure_wrapped_if_datetimelike(values)
```

Comment:
blk_values should already be EA in the relevant cases

Reply:
Indeed, removed